### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Austinati** [lukeaschenbrenner/TxtNet-Browser#37]
@lukeaschenbrenner Is there an alternative number for TxtNet-Browser?

**Comment by Austinati** [comment]
It seems that the number won't respond?

**Comment by Austinati** [comment]
Or is the server still down?

**Comment by RainbowShatter** [comment]
maybe use yours? idk, mine had slow internet so yeah

**Comment by RainbowShatter** [comment]
for me it says MS generic failure every time I click in one of the links

**Comment by Austinati** [comment]
@RainbowShatter I'm considering using my own... 😂

**Comment by RainbowShatter** [comment]
it's ok

Em 5 de agosto de 2024 00:36:55 BRT, Austinati ***@***.***> escreveu:
>@RainbowShatter I'm considering using my own... 😂
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/lukeaschenbrenner/TxtNet-Browser/issues/37#issuecomment-2268108629
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>
Sent from a fellow music maker

**Comment by lukeaschenbrenner** [comment]
Sorry for the lack of communication and development all. It's been a little bit of a stressful last few months for me but everything's sorted out and I'm slowly working on the app again. The server provided (913 area code) will be sunset pretty soon, and I'm looking into alternatives with more reliable uptime (up to now, I've only been able to hit about 25% uptime but I'm looking for closer to 95%.) For now I recommend selfhosting.

**Comment by Austinati** [comment]
@lukeaschenbrenner Any updates on this project?

**Comment by RainbowShatter** [comment]
sorry for not responding, but maybe it is because i'm in Brazil? i am not sure...

**Comment by Austinati** [comment]
Running my own server doesn't seem to help.

**Comment by Austinati** [comment]
@RainbowShatter Is it working for you?

**Comment by RainbowShatter** [comment]
Actually no, I have uninstalled this from a long time ago. Lemme see if it works, plus the server from this github...

**Comment by Austinati** [comment]
@lukeaschenbrenner is not responding and has not provided a contact method.

**Comment by Austinati** [comment]
The server has responded with "SMS Generic failure, close app and upload logs" and does not send-out.

Is this right?

**Comment by Austinati** [comment]
Some error stuff?

https://github.com/lukeaschenbrenner/TxtNet-Browser/blob/master/app/src/main/java/com/txtnet/txtnetbrowser/receiver/SmsSentReceiver.java

**Comment by Austinati** [comment]
Finally got the server to respond.

**Comment by RainbowShatter** [comment]
What the server responded? Something like "Hello, my name is..."? ;)

**Comment by Austinati** [comment]
It respond with 3 Process starting then the content. But I am no longer getting the Device not compatible error. To get rid of this, I had to download 1.0 granted all permissions Then upgrade it to the latest version. Then set up shzuku.

**Comment by Austinati** [comment]
Don't know if it works in the app.

**Comment by Austinati** [comment]
I'm trying to find a cheap alternative to do this...

**Comment by Austinati** [comment]
Do you have a contact method I can use to contact you?

**Comment by RainbowShatter** [comment]
```shatterrainbow@gmail.com```

i barely have spams

**Comment by lukeaschenbrenner** [comment]
Hey @Austinati and @RainbowShatter , sorry for the delay. "SMS Generic Failure" is a documented issue #3 and seems to result from being on a phone released before 2023 and a carrier that is/was previously CDMA (for the US, this would be Verizon). A weird bug in Android's base meant that fancy characters were converted to UCS-2 over CDMA. Since this app avoids UCS-2 entirely and uses special characters, it simply won't run. If I get a ton of people with the same issue, I may look into seeing what would be possible with supporting UCS-2. The problem with UCS-2 is that there are a TON of unicode characters out there, and a large portion of them are not supported or would do weird things when placed together. The alternative would be to support ASCII (or Base64)? over GSM 03.38 which I think these phones can handle.
Sorry for the bad news!

**Comment by Austinati** [comment]
@lukeaschenbrenner Do you have an active email I can contact you by?

**Comment by lukeaschenbrenner** [comment]
@Austinati Yes, I just realized I had my GitHub email misconfigured so no emails were coming through. I've fixed that now. I'm attaching it as an image below instead of simply typing it because I have had issues in the past with web scraper bots finding it and sending me spam. If you can't read it, let me know.

![](https://raw.githubusercontent.com/lukeaschenbrenner/lukeaschenbrenner/main/electronic-mail-address.gif)
